neondatabase/website
# chore: preload rive wasm locally

This pull request brings local rive wasm file preload instead of loading from CDN.

<details>
<summary>screenshots</summary>

**before**

![image](https://github.com/neondatabase/website/assets/20288937/2cfcdd0c-8615-4b65-859d-fd3783f33e4b)

**after**

![image](https://github.com/neondatabase/website/assets/20288937/0ba082c1-d4d7-4341-86ea-6656f535c9c1)


</details>

Steps to check:

- open [preview](https://neon-next-git-preload-rive-file-neondatabase.vercel.app/branching)
- make sure that rive.wasm file preloads locally
- make sure that og image generation working as expected ([docs](https://neon-next-git-preload-rive-file-neondatabase.vercel.app/docs/introduction), [og image](https://neon-next-git-preload-rive-file-neondatabase.vercel.app/docs/og?title=TmVvbiBkb2N1bWVudGF0aW9u&category=) sample)
